cpp/log/cert.h

```diff
@@ -52,9 +58,13 @@ class Cert {
   // underlying copy succeeded.
   Cert* Clone() const;
 
-  // Frees the old X509 and attempts to load anew.
+  // Frees the old X509 and attempts to load a new one.
   Status LoadFromDerString(const std::string& der_string);
 
+  // Frees the old X509 and attempts to load from BIO in DER form. Caller
+  // still owns the BIO afterwards.
+  Status LoadFromDerBio(BIO *bio_in);
```

Comment:
All of these should be fixed now. BTW are you sure about the log indents?
I've got Eclipse configured with Google style and it disagrees. It's not
always right though.

Martin


On 3 August 2015 at 20:08, Pierre Phaneuf <notifications@github.com> wrote:

> In cpp/log/cert.h
> <https://github.com/google/certificate-transparency/pull/859#discussion_r36119256>
> :
>
> >    Status LoadFromDerString(const std::string& der_string);
> >
> > +  // Frees the old X509 and attempts to load from BIO in DER form. Caller
> > +  // still owns the BIO afterwards.
> > +  Status LoadFromDerBio(BIO *bio_in);
>
> Style: * with type.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/certificate-transparency/pull/859/files#r36119256>
> .
>